src/utils/url.js

```diff
@@ -156,3 +156,6 @@ export const autoCompleteUrl = (
   `${hasProtocol(value) ? '' : protocol}${value || 'your-org'}${
     value.indexOf('.') === -1 ? append : !value.match(/.+\..+\.+./g) ? shortAppend : ''
   }`;
+
+export const checkUrlValidity = (url: string) =>
```

Comment:
It is better that you split the validity check and the error result. Thus this function should return true / false depending on if the url is valid.

Reply:
This function is a great candidate for a unit-test. You should demonstrate via a test both that it catches invalid urls and that it does not incorrectly catch valid urls.